client/src/app/app.module.ts

```diff
@@ -39,6 +39,7 @@ import { HeaderComponent } from './components/header/header.component';
 import { LoginComponent } from './components/login/login.component';
 import { LoggedInComponent } from './components/logged-in/logged-in.component';
 import { httpInterceptorProviders } from './interceptors';
+import { InputGeoComponent } from './components/input-geo/input-geo.component';
```

Comment:
Maybe rename it to `LocationAutoComplete` because that's essentially what it does?

Reply:
Done